Fix Workflow testing to pass

```sh
================================================================ short test summary info ================================================================
FAILED tests/test_collection.py::test_microblog_post_content_is_content - TypeError: MarkdownPageParser.parse() got an unexpected keyword argument 'page'
ERROR tests/test_collection.py::test_microblog_site_page - TypeError: MarkdownPageParser.parse() got an unexpected keyword argument 'page'
ERROR tests/test_collection.py::test_microblog_site_loads_theme - TypeError: MarkdownPageParser.parse() got an unexpected keyword argument 'page'
ERROR tests/test_collection.py::test_microblog_archive_uses_microblog_theme - TypeError: MarkdownPageParser.parse() got an unexpected keyword argument 'page'
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -9,7 +9,7 @@ description = "Microblog Parser and Collection for Render Engine"
 readme = "README.md"
 
 dependencies = [
-    "render-engine >= 2024.1.1a1",
+    "render-engine >= 2024.1.1a2",
     "jinja2",
     "render_engine_markdown",
 ]

--- a/render_engine_microblog/collection.py
+++ b/render_engine_microblog/collection.py
@@ -8,7 +8,6 @@ from .themes import microblog_theme
 class MicroBlogPost(Page):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.content = self._content
 
     @property
     def _slug(self):

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -2,6 +2,7 @@ import datetime
 
 import pytest
 from render_engine import Site
+
 from render_engine_microblog import MicroBlog
 from render_engine_microblog.collection import MicroBlogPost
 

--- a/tests/test_collection.py
+++ b/tests/test_collection.py
@@ -15,7 +15,7 @@ def test_microblog_post_slug_is_timestamp(microblog_post):
 def test_microblog_post_content_is_content(microblog_collection):
     page = list(microblog_collection)[0]
     assert page._slug == "201801010000"
-    assert "<p>Hello World</p>" in page.content
+    assert "<p>Hello World</p>" in page._content
 
 
 def test_microblog_site_page(microblog_site):
@@ -37,8 +37,7 @@ def test_microblog_site_archive(microblog_site):
 
 
 def test_microblog_site_loads_theme(microblog_site):
-    print(microblog_site.engine.list_templates())
-    assert "microblog.html" in microblog_site.engine.list_templates()
+    assert "microblog.html" in microblog_site.theme_manager.engine.list_templates()
 
 
 def test_microblog_archive_uses_microblog_theme(microblog_site):

